src/App.js

```diff
@@ -9,14 +9,48 @@ import { GlobalStyle } from "./App.styles";
 import Home from "./Pages/Home/Home";
 import Navbar from "./Components/Navbar/Navbar";
 
-export default function App() {
-  return (
-    <Router>
-      <GlobalStyle />
-      <Navbar />
-      <Routes>
-        <Page exact path="/" component={Home} />
-      </Routes>
-    </Router>
-  );
+const darkTheme = {
+  main: "hsl(227, 13%, 14%)",
+  secondary: "hsl(220, 11%, 11%)",
+  textColor: "hsl(180, 8%, 97%)",
+};
+
+const lightTheme = {
+  main: "hsl(0, 0%, 98%)",
+  secondary: "hsl(0, 0%, 100%)",
+  textColor: "hsl(222, 10%, 19%)",
+};
+
+export default class App extends React.Component {
+  state = {
+    isThemeDark: true,
+  };
+
+  handleTheme = () => {
+    this.setState({ isThemeDark: !this.state.isThemeDark });
+    this.setInStorage();
+  };
+
+  setInStorage = () => {
+    localStorage.setItem("isThemeDark", JSON.stringify(this.state.isThemeDark));
+  };
+
+  render() {
+    const currentTheme =
+      JSON.parse(localStorage.getItem("isThemeDark")) === null
+        ? this.state.isThemeDark
+        : JSON.parse(localStorage.getItem("isThemeDark"));
```

Comment:
you want to do this kind of stuff in componentDid mount, treat this like an api call